src/test/scala/org/kibanaLoadTest/deploy/DeleteAll.scala

```diff
@@ -0,0 +1,60 @@
+package org.kibanaLoadTest.deploy
+
+import org.kibanaLoadTest.helpers.{CloudEnv, CloudHttpClient}
+import org.slf4j.{Logger, LoggerFactory}
+
+object DeleteAll {
+  object Scope extends Enumeration {
+    val ALL = Value("all")
+    val TEST = Value("test")
+  }
+  val logger: Logger = LoggerFactory.getLogger("deploy:DeleteAll")
+
+  def main(args: Array[String]): Unit = {
+    val scope = Scope.withName(System.getProperty("scope", "test"))
+    val env = CloudEnv.withName(System.getProperty("env", "staging"))
+    val cleanItems = collection.mutable.Map[String, String]()
+
+    val cloudClient = new CloudHttpClient(env)
+    val deployments = cloudClient.getDeployments
+    logger.info(
+      s"Found ${deployments.size} running deployments on $env environment"
+    )
+    if (deployments.isEmpty) {
+      return
```

Comment:
Do we wanna log out that there is nothing to delete?  Perhaps this logging is already occurring?

Reply:
Good, adding extra logging